### PR TITLE
smol grammar changes to README.md

### PR DESCRIPTION
**Pull request by zahash**
smol grammar changes to README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,22 +103,21 @@ by running it with the `--help` flag or reading the [rustc dev guide][rustcguide
...
 
 [msys2]: https://www.msys2.org/
 
-1. Grab the latest [MSYS2 installer][msys2] and go through the installer.
+1. Download the latest [MSYS2 installer][msys2] and go through the installer.
 
-2. Run `mingw32_shell.bat` or `mingw64_shell.bat` from wherever you installed
-   MSYS2 (i.e. `C:\msys64`), depending on whether you want 32-bit or 64-bit
+2. Run `mingw32_shell.bat` or `mingw64_shell.bat` from the MSYS2 installation
+   directory (i.e. `C:\msys64`), depending on whether you want 32-bit or 64-bit
```
Comment (jyn514) [inline README.md:120]: while you're fixing typos, this existing one is my pet peeve
```suggestion
   directory (e.g. `C:\msys64`), depending on whether you want 32-bit or 64-bit
```

File: README.md [hunk 2]
```diff
@@ -168,7 +167,7 @@ shell with:
 python x.py build
 ```
 
-Currently, building Rust only works with some known versions of Visual Studio. If
+Right now, building Rust only works with some known versions of Visual Studio. If
```
Comment (jyn514) [inline README.md:170]: why this change?

File: README.md [hunk 3]
```diff
@@ -225,7 +224,7 @@ the ABI used. I.e., if the ABI was `x86_64-pc-windows-msvc`, the directory will
 
 Since the Rust compiler is written in Rust, it must be built by a
 precompiled "snapshot" version of itself (made in an earlier stage of
-development). As such, source builds require a connection to the Internet, to
+development). As such, source builds require an Internet connection to
```
Comment (jyn514) [inline README.md:227]: why this change?

File: README.md (hunk 1 above)
Comment (jyn514) [inline README.md:120]: this seems like a clear improvement, thanks :)

**Review by jyn514 — COMMENTED** [review]
Your MSYS2 change seems like a clear improvement :) I'm not sure that the other changes are actually easier to read though.

File: README.md (hunk 2 above)
Comment (zahash) [reply README.md:170]: its not necessary, but i find "Right now" simpler than "Currently"

File: README.md (hunk 3 above)
Comment (zahash) [reply README.md:227]: its just simpler to say "Internet connection" than "connection to the Internet"

**Comment by jyn514** [comment]
Ok, none of these changes seem incorrect and I buy that they're easier to understand for non-native speakers. Thanks!

@bors r+ rollup

File: README.md (hunk 2 above)
Comment (Nilstrieb) [reply README.md:170]: I think "currently" sounds better, "right now" sounds too much like something very temporary that might change at any second too me, but this has been a problem for quite some time I think

Comment (zahash) [reply README.md:170]: i think its the opposite. "i'm not currently dating anyone" but you might in the future. "my dog isn't currently house-trained" but it might be soon.

right now just means at this present moment. it doesn't assume anything about the duration.

what do you think?

File: README.md [hunk 4]
```diff
@@ -32,7 +32,7 @@ The `x.py` command can be run directly on most systems in the following format:
 
 This is how the documentation and examples assume you are running `x.py`.
 
-Systems such as Ubuntu 20.04 LTS do not create the necessary `python` command by default when Python is installed that allows `x.py` to be run directly. In that case you can either create a symlink for `python` (Ubuntu provides the `python-is-python3` package for this), or run `x.py` using Python itself:
+Systems such as Ubuntu 20.04 LTS do not create the necessary `python` command by default when Python is installed that allows `x.py` to be run directly. In that case, you can either create a symlink for `python` (Ubuntu provides the `python-is-python3` package for this) or run `x.py` using Python itself:
```
Comment (danielhenrymantilla) [inline README.md:35]:
```suggestion
Systems such as Ubuntu 20.04 LTS do not create the necessary `python` command by default when Python is installed that allows `x.py` to be run directly. In that case, you can either create a symlink for `python` (Ubuntu provides the `python-is-python3` package for this), or run `x.py` using Python itself:
```

  - by virtue of being an Oxford comma, it is valid (but not necessary);
  - having a comma after a parenthesis is gives the reader a much needed pause.

So I'd rather keep it🙂

**Comment by jyn514** [comment]
@bors r-

https://github.com/rust-lang/rust/pull/101684#discussion_r967864613 seems reasonable, can you please fix it?

File: README.md (hunk 4 above)
Comment (zahash) [reply README.md]: done

**Comment by zahash** [comment]
i think i made all the suggested changes. but i see that the pr is waiting on author. is there anything left for me to do?

**Comment by jyn514** [comment]
@zahash nope :) when you're done making changes you can use

@rustbot author

to say it's ready for review again.
